src/deskolem.ml

```diff
@@ -0,0 +1,28 @@
+open Core
+open Terms
+
+let rec subst_inv : term -> sym -> term Bindlib.var -> term =
+  fun t s x ->
+  match t with
```

Comment:
You should start by `let h, ts = get_args t in match h with ...`

Reply:
Done in be1e25262b42f59c76a195f12f516d73002f6dac.